system/autoware_mrm_handler/src/mrm_handler/mrm_handler_core.cpp

```diff
@@ -112,6 +114,21 @@ void MrmHandler::onOperationModeAvailability(
   is_emergency_holding_ = (emergency_duration > param_.timeout_emergency_recovery);
 }
 
+void MrmHandler::publishTurnIndicatorCmd()
+{
+  using autoware_vehicle_msgs::msg::TurnIndicatorsCommand;
+  TurnIndicatorsCommand msg;
+
+  msg.stamp = this->now();
+  if (param_.turning_hazard_on.emergency && isEmergency()) {
```

Comment:
Could you add `turning_indicator_on` parameter instead of using `turning_hazard_on`?